package.json

```diff
@@ -30,6 +30,9 @@
     "type": "git",
     "url": "git://github.com/nextbigsoundinc/imagely.git"
   },
+  "engines": {
+    "node": ">=6.4.0"
```

Comment:
I believe async/await is only available in Node 7.10.x+: https://node.green/#ES2017-features-async-functions

Reply:
Awesome, thank you. Was looking at the min req for puppeteer

Will update

Reply:
Nevermind, we're using Babel to transpile so this is moot